tokio/src/runtime/driver.rs

```diff
@@ -82,10 +82,11 @@ impl Handle {
     }
 
     cfg_io_driver! {
+        #[track_caller]
         pub(crate) fn io(&self) -> &crate::runtime::io::Handle {
             self.io
                 .as_ref()
-                .expect("A Tokio 1.x context was found, but I/O is disabled. Call `enable_io` on the runtime builder to enable I/O.")
+                .expect("A Tokio 1.x context was found, but IO is disabled. Call `enable_io` on the runtime builder to enable IO.")
```

Comment:
There is an existing test that expected `IO` to be spelled this way. Instead of updating the test, I changed the message back to what it was before.